libcore/test/catch2/geometry/LineTest.cpp

```diff
@@ -231,6 +231,20 @@ TEST_CASE("geometry/Line/IsInLineSegment", "[geometry][Line][IsInLineSegment]")
     std::random_device rd;
     std::mt19937 mt(rd());
 
+
+    SECTION("Issue762")
+    {
+        Point point{4.00026, 0.};
+        Line shortLine{{4., -0.4}, {4., 0.4}};
+        bool isInShort = shortLine.IsInLineSegment(point);
+
+        Line longLine{{4., -1.95}, {4., 1.95}};
+        bool isInLong = longLine.IsInLineSegment(point);
+
+        REQUIRE(isInShort);
+        REQUIRE(isInLong);
+    }
+
```

Comment:
Could you rewrite the test, for multiple of this cases? Lines varying in length, but the distance from the line stays the same. So should be the result.

Reply:
done.